docs/tutorials/qcnn_multiworker/inference/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 The TensorFlow Quantum Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+FROM tensorflow/tensorflow:2.3.0
+
+RUN pip install tensorflow-quantum scikit-learn google-cloud-storage
+
+COPY qcnn_inference.py ./
```

Comment:
In order to do multiworker profiling, don't we need tf 2.4.1 image and tfq-nightly ?
Also don't think we need scikit-learn.... do we ? lol